test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,27 @@
+contract C {
+  function() external [] s0;
+  function() external [] s1;
+  function copyDynamicallySizedStorageArray() public {
+    s0 = s1;
+  }
```

Comment:
Can you also add the same test for internal functions? Can be in the same contract.

Reply:
In the PR , for external functions, we're trying two cases
1. external V external
2. external V external view

Should we do both corresponding cases for internal functions as well ? or just internal V internal would be okay ?

Reply:
I'd do both unless we already have a test for it somewhere. Better too many than too few as long as the tests are concise (the way they are now seems fine +/- my comments).